k8s/namespaces/sscs/sscs-evidence-share/sscs-evidence-share.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: sscs-evidence-share
+  namespace: sscs
+spec:
+  releaseName: sscs-evidence-share
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/sscs-evidence-share
```

Comment:
A lot more could be brought into the base chart

Reply:
Further sections added to base chart